.github/workflows/ci.yml

```
@@ -566,11 +591,24 @@ jobs:
 
       - uses: actions/download-artifact@v3
         with:
-          name: macOS-build
+          name: macOS-build-unsigned
+
+      - name: macOS arm environment
```

Typically these names have more of a verb'y sound to them like "Do X". What does this do?

```suggestion
      - name: macOS ARM environment
```

---

done.